### PR TITLE
Release v0.7.0

### PR DESCRIPTION
**Pull request by forrestfwilliams**
Update burst product naming convention and add AVO's ALOS-1 stripmap processor

### CODE REVIEW

**Comment by forrestfwilliams** [comment]
Hey @AndrewPlayer3 it looks like there are some flake8 issues. would you mind taking a look at them?

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [0.7.0]
+### Changed
+* The naming convention for the burst products has been updated.
+* InSAR stripmap workflow to support AVO's ALOS-1 processing efforts. This workflow is specific to AVO currently, and may not work for others.
+
```
Comment (jtherrmann) [inline CHANGELOG.md:14]:
```suggestion
## [0.7.0]
### Added
* InSAR stripmap workflow to support AVO's ALOS-1 processing efforts. This workflow is specific to AVO currently, and may not work for others.
### Changed
* The naming convention for the burst products has been updated.

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [0.7.0]
+### Changed
+* The naming convention for the burst products has been updated.
+* InSAR stripmap workflow to support AVO's ALOS-1 processing efforts. This workflow is specific to AVO currently, and may not work for others.
```
Comment (jtherrmann) [inline CHANGELOG.md:13]:
```suggestion
## [0.7.0]
### Added
* InSAR stripmap workflow to support AVO's ALOS-1 processing efforts. This workflow is specific to AVO currently, and may not work for others.
### Changed
* The naming convention for the burst products has been updated.
```

File: tests/test_stripmapapp.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        roi=[1,2,3,4],
+        dem_filename='',
+    )
+    template_path = config.write_template(tmp_path / 'stripmapApp.xml')
+
+    with pytest.raises(ValueError, match=r'.*not a valid step.*'):
+        run_stripmapapp('notastep', config_xml=template_path)
+
+    with pytest.raises(ValueError, match=r'^If dostep is specified, start and stop cannot be used$'):
+        run_stripmapapp('preprocess', 'startup', config_xml=template_path)
```
Comment (jtherrmann) [inline tests/test_stripmapapp.py:47]: Add blank line at end of file.
```suggestion
        run_stripmapapp('preprocess', 'startup', config_xml=template_path)

```

**Review by jtherrmann — APPROVED** [review]

**Pull request merged**